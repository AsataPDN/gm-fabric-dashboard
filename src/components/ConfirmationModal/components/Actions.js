import styled from "styled-components";

const Actions = styled.div`
  display: flex;
  flex: 0 0 auto;
  flex-direction: row;
  justify-content: space-between;
  width: 100%;
`;

export default Actions;
