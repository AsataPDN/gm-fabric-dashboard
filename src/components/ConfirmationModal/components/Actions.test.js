import React from "react";
import { shallow } from "enzyme";
import Actions from "./Actions";

describe("Actions", () => {
  it("matches snapshot", () => {
    const aActions = shallow(<Actions />);
    expect(aActions).toMatchSnapshot();
  });
});
