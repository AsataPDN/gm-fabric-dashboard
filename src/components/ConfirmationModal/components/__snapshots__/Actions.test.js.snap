// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Actions matches snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 0 0 auto;
  -ms-flex: 0 0 auto;
  flex: 0 0 auto;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  width: 100%;
}

<div
  className="c0"
/>
`;
